src/sentry/static/sentry/app/components/activity/item/avatar.jsx

```diff
@@ -43,14 +43,10 @@ const SystemAvatar = styled('span')`
   display: flex;
   justify-content: center;
   align-items: center;
-  border-radius: 100%;
-  background-color: ${p => p.theme.purple};
   width: ${p => p.size}px;
   height: ${p => p.size}px;
-  vertical-align: middle;
-  position: relative;
 `;
 
 const Logo = styled(InlineSvg)`
-  color: #fff;
+  color: ${p => p.theme.gray5};
```

Comment:
Perhaps `transparent` would be better here

Reply:
I removed the background, so it's just the sentry logo in black